### ISSUE TITLE
get server missing data

### ISSUE THREAD

**Issue by chrislittle** [CenturyLinkCloud/clc-go-cli#67]
the get server function is missing some data.  In particular:
- "isManagedOS":true
- "isManagedBackup":true

below is the output from a managed OS with Managed Backup and this data is not present.

```
{
    "ChangeInfo": {
        "CreatedBy": "chris.little.ccva",
        "CreatedDate": "2016-09-02 17:05:41",
        "ModifiedBy": "l7.api.ctlx",
        "ModifiedDate": "2016-09-02 17:25:52"
    },
    "Description": "MBK",
    "Details": {
        "AlertPolicies": [],
        "Cpu": 4,
        "CustomFields": [],
        "DiskCount": 4,
        "Disks": [
            {
                "Id": "0:0",
                "PartitionPaths": [],
                "SizeGB": 1
            },
            {
                "Id": "0:1",
                "PartitionPaths": [],
                "SizeGB": 2
            },
            {
                "Id": "0:2",
                "PartitionPaths": [],
                "SizeGB": 14
            },
            {
                "Id": "0:3",
                "PartitionPaths": [],
                "SizeGB": 10
            }
        ],
        "HostName": "ca3ccvambk01.ca3.savvis.net",
        "InMaintenanceMode": false,
        "IpAddresses": [
            {
                "Internal": "10.100.98.12",
                "Public": ""
            }
        ],
        "MemoryMB": 8192,
        "Partitions": [
            {
                "Path": "(swap)",
                "SizeGB": 0
            },
            {
                "Path": "/",
                "SizeGB": 13.655
            },
            {
                "Path": "/boot",
                "SizeGB": 0.476
            },
            {
                "Path": "/data01",
                "SizeGB": 9.718
            }
        ],
        "PowerState": "started",
        "Snapshots": [],
        "StorageGB": 27
    },
    "DisplayName": "CA3CCVAMBK01",
    "GroupId": "7d971d0159e54ecea08cffafb6c3e55b",
    "Id": "ca3ccvambk01",
    "IsTemplate": false,
    "LocationId": "CA3",
    "Name": "CA3CCVAMBK01",
    "Os": "redHat6_64Bit",
    "OsType": "RedHat Enterprise Linux 6 64-bit",
    "Status": "active",
    "StorageType": "standard",
    "Type": "standard"
}
```

**Comment by chrislittle** [comment]
I'm also finding this is missing the secondary vNIC's on guests.

```
C:\clc>clc server get --server-name DE1CCVAVERSA01
{
    "ChangeInfo": {
        "CreatedBy": "Bill.Grubbs.ccva",
        "CreatedDate": "2016-09-03 12:37:59",
        "ModifiedBy": "3d94637d9bac4609b90bab87ae1153a0",
        "ModifiedDate": "2016-09-03 14:32:36"
    },
    "Description": "Versa Networks VNF Appliance",
    "Details": {
        "AlertPolicies": [],
        "Cpu": 4,
        "CustomFields": [],
        "DiskCount": 1,
        "Disks": [
            {
                "Id": "0:1",
                "PartitionPaths": [],
                "SizeGB": 90
            }
        ],
        "HostName": "",
        "InMaintenanceMode": false,
        "IpAddresses": [
            {
                "Internal": "10.110.9.12",
                "Public": "66.155.4.143"
            }
        ],
        "MemoryMB": 4096,
        "Partitions": [],
        "PowerState": "started",
        "Snapshots": [],
        "StorageGB": 90
    },
    "DisplayName": "DE1CCVAVERSA01",
    "GroupId": "f746e65a223947e590ad9230dee624b5",
    "Id": "de1ccvaversa01",
    "IsTemplate": false,
    "LocationId": "DE1",
    "Name": "DE1CCVAVERSA01",
    "Os": "ubuntu14_64Bit",
    "OsType": "Ubuntu 14 64-bit",
    "Status": "active",
    "StorageType": "standard",
    "Type": "standard"
}
```

![nic bug](https://cloud.githubusercontent.com/assets/11097012/18473371/899bec34-798a-11e6-87c3-14fb2042f2e9.png)

**Comment by richardcase** [comment]
Fixed in v1.1.0-rc.1
